usuario/models.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+
+class Usuario(models.Model):
+    id = models.AutoField(primary_key=True)
+    nome = models.CharField(max_length=255)
+    sobrenome = models.CharField(max_length=255)
+    cpf = models.CharField(max_length=14)
+    email = models.CharField(max_length=255)
+    telefone = models.CharField(max_length=20)
+    data_inicio_moradia = models.DateField()
+    data_fim_moradia = models.DateField()
+    status = models.BooleanField()
+
+    class Meta:
+        db_table = 'usuario'
+
+    def __str__(self):
+        return 'Nome: ' + str(self.nome) + 'Sobrenome: ' \
```

Comment:
Só o nome já estaria bom. Se você for entrar no admin (Na listagem de usuários), vai perceber que não irá fazer sentido colocar todos esses campos nesse método.

Reply:
correto, corregido.